waku/v2/protocol/waku_store/waku_store.nim

```diff
@@ -457,9 +457,42 @@ proc query*(w: WakuStore, query: HistoryQuery, handler: QueryHandlerFunc) {.asyn
     return
 
   waku_store_messages.set(response.value.response.messages.len.int64, labelValues = ["retrieved"])
-
   handler(response.value.response)
 
+  
+proc findLastSeen*(list: seq[IndexedWakuMessage]): float = 
+  var lastSeenTime = float64(0)
+  for iwmsg in list.items : 
+    lastSeenTime = if iwmsg.msg.timestamp>lastSeenTime: iwmsg.msg.timestamp else: lastSeenTime 
```

Comment:
I'd also split this if-else on separate lines for clarity, right now it is a bit hard to read

Reply:
For sure!